app/controllers/projects_controller.rb

```diff
@@ -29,7 +29,7 @@ def index
           @featured_partners = SitePartner.featured
           @regular_partners = SitePartner.regular
           @site_partners = @featured_partners + @regular_partners
-          @channels = Channel.all.reject { |c| c.permalink == 'gastromotiva' || c.permalink == 'garupa' || c.permalink == 'descontinuado_garupa' }
+          @channels = Channel.visible.reject { |c| c.permalink == 'gastromotiva' || c.permalink == 'garupa' || c.permalink == 'descontinuado_garupa' }
```

Comment:
Couldn't you create an constant to replace theses conditionals?
Something like:
```ruby
PERMALINK = ['gastromotiva', 'garupa', 'descontinuado_garupa']
@channels = Channel.visible.reject { |c| PERMALINK.include? c.permalink }
```